sessionctx/variable/sysvar.go

```diff
@@ -1156,6 +1156,11 @@ var defaultSysVars = []*SysVar{
 	}, SetGlobal: func(s *SessionVars, val string) error {
 		return setTiDBTableValue(s, "tidb_enable_historical_stats", val, "Current historical statistics enable status")
 	}},
+	{Scope: ScopeGlobal, Name: TiDBHistoricalStatsMaxDuration, Value: "10m0s", Type: TypeDuration, MinValue: int64(time.Minute * 10), MaxValue: uint64(time.Hour * 24 * 365), GetGlobal: func(s *SessionVars) (string, error) {
+		return getTiDBTableValue(s, "tidb_historical_stats_max_duration", "10m0s")
+	}, SetGlobal: func(s *SessionVars, val string) error {
+		return setTiDBTableValue(s, "tidb_historical_stats_max_duration", val, "Max duration which historical stats reserved, at least 10m, in Go format.")
```

Comment:
You don't need to use `getTiDBTableValue` / `setTiDBTableValue`, since the values will automatically be saved into `mysql.global_variables`. These functions are only required for the legacy GC variables, which were stored in `mysql.tidb`.

I would like to eventually remove the `mysql.tidb` table (and these functions), but currently we need to permit rolling upgrade from any version of TiDB. So it handles the case that a TiDB 4.0 server could be part of the cluster and update the `mysql.tidb` table (and you won't have this case for a new variable).

Reply:
> You don't need to use `getTiDBTableValue` / `setTiDBTableValue`, since the values will automatically be saved into `mysql.global_variables`. These functions are only required for the legacy GC variables, which were stored in `mysql.tidb`.
> 
> I would like to eventually remove the `mysql.tidb` table (and these functions), but currently we need to permit rolling upgrade from any version of TiDB. So it handles the case that a TiDB 4.0 server could be part of the cluster and update the `mysql.tidb` table (and you won't have this case for a new variable).

I found that if I remove `get`/`set`, we cannot get this global variable "`tidb_historical_stats_max_duration`" in mysql-query `show global variables like "%duration%"`. : (

So what should I do to create a new global variable for now version?